commit d6d89b33bee20879c4cef3b7fcc9c4d42c4b6eff
Author: sds <sds>
Date:   Thu Jan 8 02:56:33 2009 +0000

    fix bug #[ 2492106 ]: CLISP 2.47/amd64 can't build "pari" module
    (new_galois_format, factor_add_primes): fix type declarations

diff --git a/modules/pari/pari.lisp b/modules/pari/pari.lisp
index d5cf89d..d5586d3 100644
--- a/modules/pari/pari.lisp
+++ b/modules/pari/pari.lisp
@@ -165,9 +165,9 @@
     (and e (foreign-value e))))
 
 ;; extern int new_galois_format;
-(def-c-var new_galois_format (:type (c-ptr int)))
+(def-c-var new_galois_format (:type int))
 ;; extern int factor_add_primes;
-(def-c-var factor_add_primes (:type (c-ptr int)))
+(def-c-var factor_add_primes (:type int))
 
 ;; extern ulong DEBUGFILES, DEBUGLEVEL, DEBUGMEM
 (def-c-var debugfiles (:name "DEBUGFILES") (:type ulong))
diff --git a/src/ChangeLog b/src/ChangeLog
index 4f386a3..410b02b 100644
